asciinema2gif

```diff
@@ -5,18 +5,18 @@
 # Halt on error.
 set -e
 
-rundir=$PWD
+readonly rundir="$(basename "${0}")"
```

Comment:
That’s because you’re trying to run it with no arguments, which shouldn’t happen, anyway. Was saving that fix for another PR (for logical separation), but [included it now](https://github.com/vitorgalvao/asciinema2gif/commit/66e7dfc897946590aa78e653244685f0e9bee9bc).